test/maketests.com

```diff
@@ -150,7 +150,8 @@ $ TEST_FILES = "BNTEST,ECTEST,ECDSATEST,ECDHTEST,IDEATEST,"+ -
 	       "EVP_TEST,EVP_EXTRA_TEST,IGETEST,JPAKETEST,SRPTEST,"+ -
 	       "ASN1TEST,V3NAMETEST,HEARTBEAT_TEST,"+ -
 	       "CONSTANT_TIME_TEST,VERIFY_EXTRA_TEST,"+ -
-               "CLIENTHELLOTEST,SSLV2CONFTEST,DTLSTEST"
+               "CLIENTHELLOTEST,SSLV2CONFTEST,DTLSTEST,"+ -
+               "BAD_DTLS_TEST,FATALERRTEST"
```

Comment:
while you're in the area, it would be nice to update the comment at line 155 to remove outdated test names.

Reply:
Uhmmmm...  I'm thinking I should remove that line, is that what you mean?

Reply:
I decided to do so.  Reconfirm?

Reply:
reconfirm.  I just wanted the outdated comment fixed, in whatever was most appropriate :)